runtime/centrifuge/src/xcm.rs

```diff
@@ -105,9 +74,10 @@ impl staging_xcm_executor::Config for XcmConfig {
 	type PalletInstancesInfo = crate::AllPalletsWithSystem;
 	type ResponseHandler = PolkadotXcm;
 	type RuntimeCall = RuntimeCall;
-	type SafeCallFilter = SafeCallFilter;
+	type SafeCallFilter = Everything;
 	type SubscriptionService = PolkadotXcm;
 	type Trader = Trader;
+	type TransactionalProcessor = FrameTransactionalProcessor;
```

Comment:
It just wraps the XCM call into a transactional layer and rolls back in case of errors: https://github.com/paritytech/polkadot-sdk/blob/851c824dc7a8e7e6db2d0c9fb29d232f2f45198a/polkadot/xcm/xcm-builder/src/transactional.rs#L23